scripts/release/getPackageType.ts

```diff
@@ -0,0 +1,50 @@
+import { PACKAGE_TYPES } from './types';
+import path from 'path';
+import fs from 'fs';
+import { base } from './const';
+
+const integrationWrapperPackages = [
+  'api-client',
+  'composables',
+  'theme'
+];
+
+const getDirectories = (source: string) =>
+  fs.readdirSync(source, { withFileTypes: true })
```

Comment:
seems like you always use`path.resolve(__dirname, `${base}${pckg}`)` so put it here and use just `const directories = getDirectories(pckg);`

Reply:
Done